src/nimibook/entries.nim

```diff
@@ -1,4 +1,4 @@
-import std / [os, strutils, strformat]
```

Comment:
This breaks it on Windows as `string.replace` is in `strutils`

Reply:
Oopsie